### ISSUE TITLE
React.createClass deprecation

### ISSUE THREAD

**Issue by bryan** [danielstocks/react-sortable#74]
Currently getting a warning about:

> Warning: A Component: React.createClass is deprecated and will be removed in version 16. Use plain JavaScript classes instead. If you're not yet ready to migrate, create-react-class is available on npm as a drop-in replacement.

A package update may be needed.

**Comment by danielravina** [comment]
I'm using it just fine with a pure function
```js
import React, { PropTypes } from 'react';
import { sortable } from 'react-sortable';

const SortableListItem = (props) => (
  <div {...props}>
    {props.children}
  </div>
);

SortableListItem.propTypes = {
  children: PropTypes.node,
};

export default sortable(SortableListItem);
```

**Comment by antonigiske** [comment]
@danielravina Would you mind showing me the rest of this example? I'm not getting it to work properly. Much appreciated.

**Comment by danielravina** [comment]
Sure. This is how you use the `SortableListItem` above
```js
class List extends React.Component {
  constructor() {
    super()
    this.state = {
      draggingIndex: null
      items = [
        'Black',
        'Blue',
        'Green',
        'Yellow'
      ]
    }
  }

  onDrag = (obj) => {
    if (!obj.draggingIndex) {
      // Item has dropped
    }

    this.setState(obj);
  }

  render() {
    this.state.items.map((item, i) => {
      <SortableListItem
        key={i}
        updateState={this.onDrag}
        items={this.state.items}
        draggingIndex={this.state.draggingIndex}
        sortId={i}
        outline="list"
      >
        {item}
      </SortableListItem>
    })
  }
}
```

**Comment by mattjbrent** [comment]
**2nd the React.CreateClass error** ... In SortableComposition.js line 54. Although the github code seems to use plain class syntax, this could perhaps be an issue with the version on NPM?

**Comment by Dharmoslap** [comment]
Fixed in 1.3.0
